profiles/video.jsonld

```diff
@@ -467,83 +468,49 @@
           "rule": "included"
         }
       ]
-    },
-    {
-      "id": "https://w3id.org/xapi/video/templates#terminated",
-      "inScheme": "https://w3id.org/xapi/video",
-      "prefLabel": {
-        "en": "Terminated Statement Template for Video Profile."
-      },
-      "verb": "https://w3id.org/xapi/video/verbs/terminated",
-      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
-      "rules": [
-        {
-          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
-          "rule": "recommended"
-        },
-        {
-          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/progress']",
-          "rule": "recommended"
-        }
-      ]
-    },
-    {
-      "id": "https://w3id.org/xapi/video/templates#basic",
-      "name": {
-        "en": "Video Interactions"
-      },
-      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
-      "rules": [{
-        "location": "verb.id",
-        "any": [
-          "https://w3id.org/xapi/video/verbs/played",
-          "https://w3id.org/xapi/video/verbs/paused",
-          "https://w3id.org/xapi/video/verbs/seeked"
-        ]
-        }]
-      }
+    }
   ],
   "patterns": [
       {
-         "id": "_:generalpattern",
+         "id": "https://w3id.org/xapi/video/patterns#generalpattern",
          "primary": true,
           "name": {
               "en": "General Pattern"
           },
           "definition": {
               "en": "This pattern describes the overall sequence of statements sent over the course of an attempt"
           },
-          "sequence": ["https://w3id.org/xapi/video/templates#initialized", "_:optionalmiddlestatements", "https://w3id.org/xapi/video/templates#terminated"]
+          "sequence": ["https://w3id.org/xapi/video/templates#initialized", "https://w3id.org/xapi/video/patterns#optionalmiddlestatements"]
       },
       {
-          "@id": "_:played-pattern",
-          "alternates": ["https://w3id.org/xapi/video/templates#generalrestrictions", "https://w3id.org/xapi/video/templates#played"]
+          "@id": "https://w3id.org/xapi/video/patterns#played-pattern",
+          "alternates": ["https://w3id.org/xapi/video/templates#played"]
```

Comment:
you can't have an alternates of length 1 (you just don't need the pattern, then).